rust/PAC/README.md

```diff
@@ -0,0 +1,33 @@
+# Ibex PAC (Peripheral Access Crate)
+A Peripheral Access Crate (PAC) for the IBex Demo System from Lowrisc. See the [svd2rust](https://docs.rs/svd2rust/0.29.0/svd2rust/) documentation for more information on how to use this crate.
+
+## How to generate
+Install the tooling
+```sh
+cargo install svd2rust
+cargo install form
+```
+Generate from the svd
+```sh
+svd2rust --target=riscv -i ../../data/ibex.svd -o ./
+```
+Split the lib.rs:  
+``` sh
+~/.cargo/bin/form -i lib.rs -o src/ && rm lib.rs
+cargo fmt
+```
+
+## How to import on your application
+1. Add this crate a dependency to the application on the .toml file.
+```
+IBEX = {path = "../../PAC"}
+```
```

Comment:
```suggestion
    ```
    IBEX = {path = "../../PAC"}
    ```
```

You can indent code blocks to make them part of list items. See [spec](https://spec.commonmark.org/0.30/#example-263).